src/components/contestants.less

```diff
@@ -0,0 +1,274 @@
+@import "../main";
+
+
+.contestants {
+  .blue-section-wrapper {
+    .blue-section-wrapper (@img_path: "../../assets/img/contestants.png");
+  }
+
+  .blue-section {
+    .blue-section(@min-height: 500px);
```

Comment:
Avem în cod variabila `section-min-height` definita in `variables.less` pe care o folosim o singura data la `.blue-section (@min-height: @section-min-height * 2)`.

Sterge-o de tot si fa-i `.blue-section (@min-height: 500px)`.

Apoi schimba de peste tot de unde ai facut `.blue-section(@min-height: 500px);` in `.blue-section()`.